CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -435,6 +472,10 @@ public void setToolHome(String toolHome) {
 	public String getCtpBranchName() {
 		return ctpBranchName;
 	}
+	
+	public String getFeedbackType(){
```

Comment:
Remove getFeedbackType().

Also remove setFeedback(...).

Just keep getFeedback(). (It requires to initialize feedback variable).